delete-publication-from-scopus/src/test/java/no/sikt/nva/scopus/ScopusDeleteHandlerTest.java

```diff
@@ -0,0 +1,210 @@
+    public static final String HARDCODED_SCOPUS_IDENTIFIER_IN_DELETE_SINGLE_FILE = "0000687314";
+    public static final List<String> HARDCODED_SCOPUS_IDENTIFIERS_IN_DELETE_MULTIPLE_FILE =
+            Arrays.asList("0000687314", "0017752991", "33947568076", "85084744636");
+
+    private FakeS3Client s3Client;
+    private S3Driver s3Driver;
+    private ScopusDeleteHandler scopusDeleteHandler;
+    private FakeEventBridgeClient fakeEventBridgeClient;
+
+
+    @BeforeEach
+    public void init() {
+        s3Client = new FakeS3Client();
+        s3Driver = new S3Driver(s3Client, "ignoredValue");
+        fakeEventBridgeClient = new FakeEventBridgeClient();
+        scopusDeleteHandler = new ScopusDeleteHandler(s3Client, fakeEventBridgeClient);
+    }
+
+    @Test
+    void shouldEmitEventWithSingleScopusIdentifierWhenInputFileContainsSingleScopusIdentifier() throws IOException {
+        var scopusFile = IoUtils.stringFromResources(Path.of(DELETE_SINGLE_IDENTIFIER_FILE));
+        var uri = s3Driver.insertFile(UnixPath.of(randomString()), scopusFile);
+        var s3Event = createS3Event(uri);
+        assertThat(fakeEventBridgeClient.getRequestEntries().size(), is(equalTo(0)));
+        scopusDeleteHandler.handleRequest(s3Event, CONTEXT);
+        var actualScopusDeleteEventBodies = getScopusDeleteEventBodies();
+        assertThat(actualScopusDeleteEventBodies.size(), is(equalTo(1)));
+        assertThat(actualScopusDeleteEventBodies.get(0).getScopusIdentifier(),
+                is(equalTo(HARDCODED_SCOPUS_IDENTIFIER_IN_DELETE_SINGLE_FILE)));
+    }
+
+    @Test
+    void shouldEmitEventsWithScopusIdentifiersWhenInputFileContainsMultipleScopusIdentifiers() throws IOException {
+        var scopusFile = IoUtils.stringFromResources(Path.of(DELETE_MULTIPLE_IDENTIFIERS_FILE));
+        var uri = s3Driver.insertFile(UnixPath.of(randomString()), scopusFile);
+        var s3Event = createS3Event(uri);
+        scopusDeleteHandler.handleRequest(s3Event, CONTEXT);
+        var actualScopusDeleteEventBodies = getScopusDeleteEventBodies();
+        assertThat(actualScopusDeleteEventBodies.size(),
+                is(equalTo(HARDCODED_SCOPUS_IDENTIFIERS_IN_DELETE_MULTIPLE_FILE.size())));
+        assertThat(actualScopusDeleteEventBodies.get(0).getScopusIdentifier(),
+                is(equalTo(HARDCODED_SCOPUS_IDENTIFIERS_IN_DELETE_MULTIPLE_FILE.get(0))));
+        assertThat(actualScopusDeleteEventBodies.get(1).getScopusIdentifier(),
+                is(equalTo(HARDCODED_SCOPUS_IDENTIFIERS_IN_DELETE_MULTIPLE_FILE.get(1))));
+        assertThat(actualScopusDeleteEventBodies.get(2).getScopusIdentifier(),
+                is(equalTo(HARDCODED_SCOPUS_IDENTIFIERS_IN_DELETE_MULTIPLE_FILE.get(2))));
+        assertThat(actualScopusDeleteEventBodies.get(3).getScopusIdentifier(),
+                is(equalTo(HARDCODED_SCOPUS_IDENTIFIERS_IN_DELETE_MULTIPLE_FILE.get(3))));
```

Comment:
Are we interested in the ordering, or can we just assert that the list of identifiers in event bodies equals the list of hardcoded identifiers?